### PR TITLE
[1.0] Add Source repos for base, update, ui, preview and preview-ui

### PR DESCRIPTION
**Pull request by anphel31**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add references to the Source/SRPMS repos on PMC for CBL-Mariner 1.0 repos.
These can be used to install source packages using for example "sudo tdnf install foo.src --enablerepo=mariner-official-base-source"

###### Change Log  

- Change mariner-repos subpackages to reference source repos on PMC

###### Does this affect the toolchain?  

**YES**

###### Test Methodology

- Pipeline build id: 214975 (succeeded).

### CODE REVIEW

File: SPECS/mariner-repos/mariner-repos.spec [hunk 1]
```diff
@@ -83,6 +86,29 @@ Requires: %{name} = %{version}-%{release}
...
+%package microsoft-update-srpms
+Summary:  CBL-Mariner Update SRPMS repository.
+Group:    System Envrionment/Base
+Requires: %{name} = %{version}-%{release}
+
+%description microsoft-update-srpms
+%{summary}
+
+%package microsoft-ui-srpms
+Summary:  CBL-Mariner UI SRPMS repository.
```
Comment (jslobodzian) [inline SPECS/mariner-repos/mariner-repos.spec:106]: lets add the ui preview repo too

File: SPECS/mariner-repos/mariner-repos.spec [hunk 2]
```diff
@@ -83,6 +86,29 @@ Requires: %{name} = %{version}-%{release}
...
 
+%package microsoft-base-srpms
+Summary:  CBL-Mariner Base SRPMS repository.
+Group:    System Envrionment/Base
+Requires: %{name} = %{version}-%{release}
+
+%description microsoft-base-srpms
+%{summary}
+
+%package microsoft-update-srpms
```
Comment (jslobodzian) [inline SPECS/mariner-repos/mariner-repos.spec:97]: let's add the update preview repo too

File: SPECS/mariner-repos/mariner-repos.spec (hunk 1 above)
Comment (anphel31) [reply SPECS/mariner-repos/mariner-repos.spec:106]: sure, will add

File: SPECS/mariner-repos/mariner-repos.spec (hunk 2 above)
Comment (anphel31) [reply SPECS/mariner-repos/mariner-repos.spec:97]: sure, will add

**Review by jslobodzian — APPROVED** [review]

**Pull request merged**